lib/Icon/index.js

```diff
@@ -101,6 +101,7 @@ import warningTriangleSVG from '../../assets/icons/warning-triangle.svg';
 import warningOctogonSVG from '../../assets/icons/warning-octogon.svg';
 import infoSquareSVG from '../../assets/icons/info-square.svg';
 import integrationSVG from '../../assets/icons/integration.svg';
+import resolvedSVG from '../../assets/icons/resolve.svg';
```

Comment:
Turns out the icon for resolved wasn't using our `Icon` component, so I put it in there!